source/blog/openshift-openId-integration-with-engine-sso.html.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Integrating Kibana/Elasticsearch on top of OpenShift with oVirt Engine SSO
+author: rnori
+tags: community, documentation, howto, OpenShift, oVirt, sso
+date: 2017-04-04 11:40:00 CET
+comments: true
+published: true
+---
+
+Integrating Kibana/Elasticsearch on top of OpenShift with oVirt Engine SSO
+
+Step by step instructions on how to integrate Kibana/Elasticsearch on top of OpenShift with oVirt Engine SSO
+
+READMORE
+
+## Background
+
+The goal is to integrate Kibana/Elasticsearch on top of OpenShift with oVirt Engine SSO, so existing engine users can access Kibana/Elasticsearch without reauthentication (we don't need to maintain authentication configuration separately for oVirt Engine and Kibana/Elasticsearch) 
+
+## Prerequisites
+
+* A fully working and configured oVirt engine instance
+* A fully working and configured instance of Kibana/Elasticsearch on top of OpenShift
+
+## Installing Kibana/Elasticsearch and OpenShift backend
+
+Install Kibana/Elasticsearch/OpenShift on CentOS7 or RHEL 7.3 as described in https://github.com/ViaQ/Main/blob/master/README-mux.md
+
+## Installing oVirt Engine
+
+Setup oVirt Engine on a separate host ovirt-engine.example.com as described in https://www.ovirt.org/download/
+
+## Setting up oVirt Engine certificate on OpenShift machine
+
+Get the oVirt Engine CA as described here https://www.ovirt.org/documentation/how-to/guest-console/connect-to-spice-console-without-portal/
```

Comment:
Users could use above link or use scp command below to get oVirt CA cert into the OpenShift host.

Anyway both ways will work only if oVirt internal CA is used to HTTPS. If users provide their own CA, then steps to get CA cert would be a bit different, but not sure if it's worth mention it here.